pkgs/by-name/si/sirius/package.nix

```diff
@@ -110,11 +107,12 @@ stdenv.mkDerivation rec {
   doCheck = true;
 
   # Can not run parallel checks generally as it requires exactly multiples of 4 MPI ranks
```

Comment:
Is this still a problem with the mpiCheckPhaseHook?

Reply:
No, this is something special to Sirius as it wishes to have this multiples of 4 ranks. Idk how to ensure the appropriate number of build cores in the sandbox generally.